common/lib/xmodule/xmodule/video_module/video_module.py

```diff
@@ -375,6 +394,20 @@ def editable_metadata_fields(self):
         else:
             editable_fields.pop('source')
 
+        licensable = False
+        course_id = None
+        course = None
+        if hasattr(settings, 'FEATURES') and settings.FEATURES.get('CREATIVE_COMMONS_LICENSING', False):
+            if hasattr(self.runtime, 'modulestore'):
+                course_id = self.runtime.course_id
+                if course_id:
+                    course = self.runtime.modulestore.get_course(course_id)
```

Comment:
Why is this needed? "We need to have access to the course license and licensable settings, otherwise I'm not able to disable the edit feature for the license in the video editor." (just for context)